converter/dicom-cast/docs/workingWithPrivateLink.md

```diff
@@ -0,0 +1,24 @@
+# Configuration steps for dicomast to work with privatelink enabled dicom service
+
+1. Create a virtual network with two subnets within the same subscription and region as you would plan to create the health data services workspace 
+![Alt text](image.png)
+
+2. Provision Healthdataservice workspace, dicomserver and fhir service in the same region.
```

Comment:
nit. Health Data Services workspace, DICOM service, and FHIR service (also update throughout)